docs/migration/manual-migration.mdx

```diff
@@ -77,6 +91,24 @@ To cope with this change, you need to review all components using
   );
   ```
 
+## **[BREAKING]**: `PartialProductQuery` removal
```

Comment:
**issue:** instead of going fancy with formatting and conventions for breaking changes, I think it'd be better to find another pattern

**suggestion:** create a section for breaking change, and add `PartialProductQuery` as a sub-section

**suggestion(alt):** create a dedicated page for breaking changes, and link it from the the Manual Migration page.

Reply:
As discussed, pretty much all changes from "Manual migration" are breaking changes in fact, so I'll just remove the "BREAKING" tag.